tests/qa_automation/qa_run.pm

```diff
@@ -55,7 +55,7 @@ sub prepare_repos {
     pkcon_quit;
     if ($qa_server_repo) {
         # Remove all existing repos and add QA_SERVER_REPO
-        script_run('for i in {1..$(zypper lr| tail -n+3 |wc -l)}; do zypper -n rr $i; done; unset i', 300);
+        script_run('for ((i = 1; i <= $(zypper lr| tail -n+3 |wc -l); i++ )); do zypper -n rr $i; done; unset i', 300);
```

Comment:
Just a note, you don't need for-loop for this, zypper accepts multiple repositories as parameter
zypper rr `seq n`